### ISSUE TITLE
Tracking June 2022 FOQUS Machine Learning Plugin Improvements

### ISSUE THREAD

**Issue by bpaul4** [CCSI-Toolset/FOQUS#1038]
This issue exists to track planned, ongoing and integrated features as part of milestone EY22.4.1.J - Improvements to the ML/AI interface in FOQUS. Each improvement is assumed to encompass documentation and example expansion for new features, and is marked with the planned release:

- [x] Expanded normalization options, including support for additional formula options and user-provided custom formulations (June 2022) PR #1034

<strike>

- Development of core node functionality tests, both for ML/AI plugin and coverage gaps in non-ML/AI methods (September 2022)

- Investigate support for other Keras formats that are newer/more common than H5 such as SavedModel and JSON (September 2022)

- Investigate support for other Python machine learning tools such as PyTorch, SciKit-Learn and Seaborn (September 2022)
</strike>

- Moved September tasks to new issue
